scripts/build_maubot_yaml.py

```diff
@@ -12,8 +12,7 @@
 maubot_yaml["version"] = pyproject["tool"]["poetry"]["version"]
 maubot_yaml["license"] = pyproject["tool"]["poetry"]["license"]
 
-with open("requirements.txt", mode="r", encoding="utf8") as f:
-    maubot_yaml["dependencies"] = [line.strip() for line in f.readlines()]
```

Comment:
What does this change fix?  Have you built the plugin successfully after removing it?

Reply:
Yes, the yaml file doesn't properly build since there's no `requirements.txt` file anymore.

Reply:
This line should be producing it from the `poetry.lock`: https://github.com/jeffcasavant/MaubotTrumpTweet/blob/bd3a56e9a8e7d2673efd75b198f2117e29f3dc7d/Makefile#L19

I'll take a look & see if I can't replicate that build error and get it fixed for you.

For context - I don't think Maubot supports auto-installing dependencies yet, so the baked-in dependency list just gets ignored.  I would expect exactly what you're seeing - that the content of the dependency list doesn't matter for plugin functionality.  I'd like to keep putting it in my plugins though so that I don't have to update them if/when that feature ships in Maubot - in particularly for ones like this that depend on complex/heavy libraries like Pillow.